File: Code/Cplus/LAB03_01/LAB03_01.cpp
#include <iostream>
using namespace std;

// Function to print numbers 1 to 10 using for loop
void printFor() {
    cout << "Using for loop: ";
    for (int i = 1; i <= 10; i++) {
        cout << i << " ";
    }
    cout << endl;
}

// Function to print numbers 1 to 10 using while loop
void printWhile() {
    cout << "Using while loop: ";
    int i = 1;
    while (i <= 10) {
        cout << i << " ";
        i++;
    }
    cout << endl;
}

// Function to print numbers 1 to 10 using do-while loop
void printDoWhile() {
    cout << "Using do-while loop: ";
    int i = 1;
    do {
        cout << i << " ";
        i++;
    } while (i <= 10);
    cout << endl;
}

int main() {
    printFor();
    printWhile();
    printDoWhile();
    return 0;
}

File: Code/Python/LAB03_01.py
# Function to print numbers 1 to 10 using for loop
def print_for():
    print("Using for loop: ", end="")
    for i in range(1, 11):
        print(i, end=" ")
    print()

# Function to print numbers 1 to 10 using while loop
def print_while():
    print("Using while loop: ", end="")
    i = 1
    while i <= 10:
        print(i, end=" ")
        i += 1
    print()

# Function to print numbers 1 to 10 using do-while loop
def print_do_while():
    print("Using do-while loop: ", end="")
    i = 1
    while True:
        print(i, end=" ")
        i += 1
        if i > 10:
            break
    print()

def main():
    print_for()
    print_while()
    print_do_while()

if __name__ == "__main__":
    main()
